apiary.apib

```diff
@@ -2713,11 +2713,15 @@ Get a list of projects.
                 + source (object)
                     + type: `deal` (string)
                     + id: `5023d7c2-80d7-4d4b-b2bd-0fcaa6a1f069` (string)
+                + actuals (object)
```

Comment:
Should we add a comment next to this saying that it is only available for admins?